ci-operator/step-registry/gather/extra/jump/gather-extra-jump-commands.sh

```diff
@@ -0,0 +1,199 @@
+  queue ${ARTIFACT_DIR}/nodes/$i/heap oc --insecure-skip-tls-verify get --request-timeout=20s --raw /api/v1/nodes/$i/proxy/debug/pprof/heap
+done < ${WORKSPACE}/nodes
+
+FILTER=gzip queue ${ARTIFACT_DIR}/nodes/masters-journal.gz oc --insecure-skip-tls-verify adm node-logs --role=master --unify=false
+FILTER=gzip queue ${ARTIFACT_DIR}/nodes/masters-journal-previous.gz oc --insecure-skip-tls-verify adm node-logs --boot=-1 --role=master --unify=false
+FILTER=gzip queue ${ARTIFACT_DIR}/nodes/workers-journal.gz oc --insecure-skip-tls-verify adm node-logs --role=worker --unify=false
+FILTER=gzip queue ${ARTIFACT_DIR}/nodes/workers-journal-previous.gz oc --insecure-skip-tls-verify adm node-logs --boot=-1 --role=worker --unify=false
+
+# Snapshot iptables-save on each node for debugging possible kube-proxy issues
+oc --insecure-skip-tls-verify get --request-timeout=20s -n openshift-sdn -l app=sdn pods --template '{{ range .items }}{{ .metadata.name }}{{ "\n" }}{{ end }}' > ${WORKSPACE}/sdn-pods
+while IFS= read -r i; do
+  queue ${ARTIFACT_DIR}/network/iptables-save-$i oc --insecure-skip-tls-verify rsh --timeout=20 -n openshift-sdn -c sdn $i iptables-save -c
+done < ${WORKSPACE}/sdn-pods
+
+while IFS= read -r i; do
+  file="$( echo "$i" | cut -d ' ' -f 3 | tr -s ' ' '_' )"
+  queue ${ARTIFACT_DIR}/metrics/${file}-heap oc --insecure-skip-tls-verify exec $i -- /bin/bash -c 'oc --insecure-skip-tls-verify get --raw /debug/pprof/heap --server "https://$( hostname ):8443" --config /etc/origin/master/admin.kubeconfig'
+  queue ${ARTIFACT_DIR}/metrics/${file}-controllers-heap oc --insecure-skip-tls-verify exec $i -- /bin/bash -c 'oc --insecure-skip-tls-verify get --raw /debug/pprof/heap --server "https://$( hostname ):8444" --config /etc/origin/master/admin.kubeconfig'
+done < ${WORKSPACE}/pods-api
+
+while IFS= read -r i; do
+  file="$( echo "$i" | cut -d ' ' -f 2,3,5 | tr -s ' ' '_' )"
+  FILTER=gzip queue ${ARTIFACT_DIR}/pods/${file}.log.gz oc --insecure-skip-tls-verify logs --request-timeout=20s $i
+  FILTER=gzip queue ${ARTIFACT_DIR}/pods/${file}_previous.log.gz oc --insecure-skip-tls-verify logs --request-timeout=20s -p $i
+done < ${WORKSPACE}/containers
+
+echo "Snapshotting prometheus (may take 15s) ..."
+queue ${ARTIFACT_DIR}/metrics/prometheus.tar.gz oc --insecure-skip-tls-verify exec -n openshift-monitoring prometheus-k8s-0 -- tar cvzf - -C /prometheus .
+FILTER=gzip queue ${ARTIFACT_DIR}/metrics/prometheus-target-metadata.json.gz oc --insecure-skip-tls-verify exec -n openshift-monitoring prometheus-k8s-0 -- /bin/bash -c "curl -G http://localhost:9090/api/v1/targets/metadata --data-urlencode 'match_target={instance!=\"\"}'"
+
+wait
+
+
+EOF
+
+run_rsync() {
+  set -x
+  rsync -PazcOq -e "ssh -q -o StrictHostKeyChecking=false -o UserKnownHostsFile=/dev/null -i ${SSH_PRIV_KEY_PATH}" "${@}"
+  set +x
+}
+
+run_ssh() {
+  set -x
+  ssh -q -o StrictHostKeyChecking=false -o UserKnownHostsFile=/dev/null -i "${SSH_PRIV_KEY_PATH}" "${@}"
+  set +x
+}
+
+
```

Comment:
gate `if [[ ! -s ${SHARED_DIR}/jump-host.txt" ]]`

Reply:
Also in the other files.

Reply:
Any reason why we would do this? I don't want to skip silently.

Reply:
This will prevent `ssh` from failing when `${REMOTE}` is an empty string and throwing a possibly misleading error message. Better to gate here with a clear error of why.

Reply:
added with https://github.com/openshift/release/pull/9553/commits/eec6cc05ff781033e22518669ccc41af4cd5dc0a